docs/EpistemicConstraints.md

```diff
@@ -75,23 +75,23 @@ Is satisfied if `field` is a string containing the regular expression expressed
 { "field": "name", "is": "ofLength", "value": 5 }
 ```
 
-Is satisfied if `field` is a string whose length exactly matches `value`.
+Is satisfied if `field` is a string whose length exactly matches `value`, must be a whole number.
```

Comment:
I think it would add confusion to imply that 10.0 is also accepted. Normally you wouldn't expect this fraction to be included, and calling out that it is ok seems superfluous. We know it will handle it, but i'm not sure it adds anything to the documentation.

Reply:
I agree with you, following discussion.